### PR TITLE
Add support for new session extension handling in Electron 9

### PR DESCRIPTION
**Pull request by develerik**
electron 9 deprecates the use of BrowserWindow extension handling so I implemented the new way for handling extensions

### CODE REVIEW

**Comment by sindresorhus** [comment]
Can you share a link to docs/announcement about the deprecation?

File: index.js [hunk 1]
```diff
@@ -54,13 +54,21 @@ function inspectElements() {
 
 const addExtensionIfInstalled = (name, getPath) => {
 	const isExtensionInstalled = name => {
+		if (session.defaultSession.getAllExtensions) {
```
Comment (sindresorhus) [inline index.js:57]: Add a comment that mentions that this code-path is for Electron 9 and later. Same with the other place.

File: index.js [hunk 2]
```diff
@@ -54,13 +54,21 @@ function inspectElements() {
...
+			return {}.hasOwnProperty.call(session.defaultSession.getAllExtensions(), name);
+		}
+
 		return BrowserWindow.getDevToolsExtensions &&
```
Comment (sindresorhus) [inline index.js:61]: Add a TODO comment to remove this when targeting Electron 9. Same with the other place.

**Comment by develerik** [comment]
> Can you share a link to docs/announcement about the deprecation?

The deprecation is documented here:
- https://www.electronjs.org/docs/api/browser-window#browserwindowgetdevtoolsextensions-deprecated
- https://www.electronjs.org/docs/api/browser-window#browserwindowadddevtoolsextensionpath-deprecated

**Comment by sindresorhus** [comment]
Thanks :)

**Pull request merged**